test/plugin/test_in_watch_process.rb

```diff
@@ -25,7 +25,8 @@ def test_configure
     assert_equal ['apache', 'mycron'], d.instance.lookup_user
   end
 
-  def test_emit
+  def test_unixlike
+    return if Fluent.windows?
```

Comment:
`omit "EXPLAIN THE REASON" if Fluent.windows?` is better if the test must be skipped.

Reply:
Thank you for the advice.
I have fixed this in https://github.com/daipom/fluent-plugin-watch-process/pull/3/commits/0fcd97df35e7d90d9dd2953bb4ad056f7003deaf.